src/components/common/table/table.js

```diff
@@ -119,15 +179,84 @@ class Table extends Component{
                         <i onClick={this.props.togleAddEmployeeModal} className="fa fa-user-plus"></i>
                     </div>
                 </div>
+                }
+                
+                {this.state.currentOpenedRowId !== null && 
+                
+                <Modal
+                    key={1}
+                    open={this.state.opinionModal}
+                    classNames={{ modal: "Modal Modal-add-owner" }}
+                    contentLabel="Employee opinion modal"
+                    onClose={() => this.setState({opinionModal: !this.state.opinionModal})}>
+                    
+                    <div className="opinion-container">
+                        <header>
+                            <h3 className="section-heading">{this.state.modalType ? 
+                            `Dodaj opinie o pracowniku ${this.props.items[this.state.currentOpenedRowId].employeeId}` : "Lista opini o pracowniku"}</h3>
+                        </header>
+                        
+                        {this.state.modalType ? 
```

Comment:
Bad approach.
You should define a const, like: ` const modalType=this.state.modalType`.
It's good practice for React developers. This should be ultimately changed everywhere.